applications/luci-app-usteer/root/usr/share/luci/menu.d/luci-app-usteer.json

```diff
@@ -0,0 +1,30 @@
+{
+  "admin/usteer/": {
+    "title": "Usteer",
+    "order": 60,
+    "action": {
+      "type": "firstchild"
+    },
+    "depends": {
+      "acl": [ "luci-app-usteer" ]
+    }
+  },
+
+  "admin/usteer/settings": {
+    "title": "Settings",
+    "order": 1,
+    "action": {
+      "type": "view",
+      "path": "usteer/settings"
+    }
+  },
+
+  "admin/usteer/status": {
+    "title": "Status page",
+    "order": 2,
+    "action": {
+      "type": "view",
+      "path": "usteer/status"
```

Comment:
Other packages, like statistics, have first the "view" and last the "config". Maybe is better to do it in the same order to maintain coherence. I don't know if there is some "rule" about this.

Reply:
im fine either way. And indeed not sure what the "rule" is. Anybody else have an opinion?

Reply:
i swapped them over for now